<ui version="4.0" >
 <author>Tomeu Borras Riera</author>
 <class>BalanceBase</class>
 <widget class="QWidget" name="BalanceBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>490</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>650</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Balance de cuentas</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../bulmages.qrc" >:/BulmaCont32x32/images/png/i_balance.xpm</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>3</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>3</number>
     </property>
     <property name="spacing" >
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="pixmapLabel1" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../bulmages.qrc" >:/BulmaCont32x32/images/png/i_balance.xpm</pixmap>
       </property>
       <property name="scaledContents" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_3" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>3</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>3</number>
        </property>
        <property name="spacing" >
         <number>3</number>
        </property>
        <item row="0" column="2" >
         <widget class="QToolButton" name="mui_filtrar" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Filtrar balance</string>
          </property>
          <property name="whatsThis" >
           <string>Filtrar balance</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages.qrc" >:/Genericos32x32/images/png/i_find.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="shortcut" >
           <string>Ctrl+F</string>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="5" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>52</width>
            <height>25</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="4" >
         <widget class="QToolButton" name="mui_actualizar" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Actualizar balance</string>
          </property>
          <property name="whatsThis" >
           <string>Actualizar balance</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages.qrc" >:/Genericos32x32/images/png/i_reload.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="shortcut" >
           <string>F5</string>
          </property>
          <property name="checkable" >
           <bool>false</bool>
          </property>
          <property name="checked" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QToolButton" name="mui_imprimir" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Imprimir balance</string>
          </property>
          <property name="whatsThis" >
           <string>Imprimir balance</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages.qrc" >:/Genericos32x32/images/png/i_filePrint.xpm</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="shortcut" >
           <string>Ctrl+P</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <widget class="QToolButton" name="mui_configurar" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Configurar</string>
          </property>
          <property name="whatsThis" >
           <string>Configurar</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages.qrc" >:/Genericos32x32/images/png/i_packageSettings.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="shortcut" >
           <string>Ctrl+B</string>
          </property>
          <property name="checkable" >
           <bool>false</bool>
          </property>
          <property name="checked" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>3</number>
      </property>
      <property name="spacing" >
       <number>3</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="TextLabel1_2_2" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Fecha inicial:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="TextLabel1_3" >
        <property name="text" >
         <string>Cuenta inicial:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>241</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" >
       <widget class="BusquedaFecha" native="1" name="m_fechainicial1" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2" >
       <widget class="BusquedaCuenta" native="1" name="m_codigoinicial" >
        <property name="minimumSize" >
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="TextLabel1_2_2_2" >
        <property name="text" >
         <string>Fecha final:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLabel" name="TextLabel1_3_2" >
        <property name="text" >
         <string>Cuenta final:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>241</width>
          <height>23</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" >
       <widget class="BusquedaFecha" native="1" name="m_fechafinal1" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="BusquedaCuenta" native="1" name="m_codigofinal" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="textLabel1_2" >
        <property name="text" >
         <string>Nivel:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>241</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0" >
       <widget class="QComboBox" name="combonivel" >
        <property name="minimumSize" >
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>261</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0" >
       <widget class="QCheckBox" name="checksuperiores" >
        <property name="text" >
         <string>Balance jerárquico</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="mui_list" >
     <column>
      <property name="text" >
       <string>1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>GroupBox</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>3</number>
      </property>
      <property name="spacing" >
       <number>3</number>
      </property>
      <item row="0" column="9" >
       <widget class="QLineEdit" name="totalsaldo" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="palette" >
         <palette>
          <active>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>238</red>
              <green>238</green>
              <blue>238</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>238</red>
              <green>238</green>
              <blue>238</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font" >
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="5" >
       <widget class="QLineEdit" name="totaldebe" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="palette" >
         <palette>
          <active>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>238</red>
              <green>238</green>
              <blue>238</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>238</red>
              <green>238</green>
              <blue>238</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font" >
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="8" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Saldo:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7" >
       <widget class="QLineEdit" name="totalhaber" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="palette" >
         <palette>
          <active>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>238</red>
              <green>238</green>
              <blue>238</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>238</red>
              <green>238</green>
              <blue>238</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font" >
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="6" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Haber:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLineEdit" name="totalsaldoant" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="palette" >
         <palette>
          <active>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>238</red>
              <green>238</green>
              <blue>238</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>238</red>
              <green>238</green>
              <blue>238</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font" >
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="TextLabel2" >
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <property name="text" >
         <string>Saldo anterior:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Debe:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>200</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="3" />
 <customwidgets>
  <customwidget>
   <class>BusquedaFecha</class>
   <extends>QWidget</extends>
   <header>../../bulmalib/src/busquedafecha.h</header>
  </customwidget>
  <customwidget>
   <class>BusquedaCuenta</class>
   <extends>QWidget</extends>
   <header>busquedacuenta.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mui_imprimir</tabstop>
  <tabstop>mui_filtrar</tabstop>
  <tabstop>mui_configurar</tabstop>
  <tabstop>mui_actualizar</tabstop>
  <tabstop>combonivel</tabstop>
  <tabstop>checksuperiores</tabstop>
  <tabstop>mui_list</tabstop>
 </tabstops>
 <resources>
  <include location="../../bulmages.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>mui_filtrar</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>160</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel" >
     <x>152</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
